### ISSUE TITLE
Pikmin 4 frame Drops on Steamdeck

### ISSUE THREAD

**Issue by Conan179** [yuzu-emu/yuzu#11138]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Affected Build(s)

Mainline 1504

### Description of Issue

When I play Pikmin 4 it often happens that the frame rate goes down.

### Expected Behavior

Stable frame rate.

### Reproduction Steps

https://www.dropbox.com/s/yjqoklj2zs0ltab/Decky-Recorder_2023-07-23_18-45-04.mp4?dl=0

### Log File

[yuzu_log.txt](https://github.com/yuzu-emu/yuzu/files/12139166/yuzu_log.txt)

### System Configuration

https://de.wikipedia.org/wiki/Steam_Deck#Spezifikationen

**Comment by relu419** [comment]
I can confirm random slowdown with Desktop. Using Vulkan, CPU auto and GPU high accuracy. The game in general runs fine.

**Comment by DarkwingChao** [comment]
Having the same slowdown also. GPU high accuracy, handheld mode, most settings at default. Is anyone on PC having this issue, or is it steam deck specific?

**Comment by relu419** [comment]
> Having the same slowdown also. GPU high accuracy, handheld mode, most settings at default. Is anyone on PC having this issue, or is it steam deck specific?

Same for PC, as I said. Since Steam Deck is a PC, same error is normal.

**Comment by gfrhj** [comment]
Will also add that it gets worse the longer the game runs. It usually goes back to full speed in caves.

**Comment by Jickes1** [comment]
Seems to be the same for me. I’ve been trying to turn on/off random settings but to no avail. Tried setting GPU accuracy to extreme but that did not change anything. Seems like a caching issue of some form.

**Comment by tagg7** [comment]
This also happens on regular desktop if you play long enough. I had assumed it was some form of memory leak in that it gets progressively worse.

**Comment by rahmatrmdn** [comment]
same here

**Comment by anyoneyun** [comment]
Same issue here

**Comment by hambo325** [comment]
Same issue here

**Comment by otelo2** [comment]
While it usually goes back to full speed in caves, in the bigger caves it also starts to slow down if left running for a long time, and when going out of the caves the main world is still at 1 fps for long periods of time

**Comment by hambo325** [comment]
> While it usually goes back to full speed in caves, in the bigger caves it also starts to slow down if left running for a long time, and when going out of the caves the main world is still at 1 fps for long periods of time

Yeah this is exactly what happens to me. Seems like a memory leak maybe? It definitely progressively gets worse too. First couple minutes of a day are fine, by afternoon it's a slideshow. Going in a cave makes it run full speed. If you manage to push past the slideshow and end the day. The following cutscenes and menus are also super slow. Only fix is quitting the game and ending days early to save.

**Comment by Ciraplex** [comment]
I had this same issue with my PC Intel i7-2 gen  GTX150TI  16gB ram SSD 1TB win 10
to solve the related issue and make the game playable on my garbage PC. I just changed the Vulkan option to OPen GL Other settings remain default or max. at this stage, I completed the intro part of the game including the first cave where we save the captain. with an average of 33 fpm.

**Comment by Grygon** [comment]
Also having serious issues here, surprised there isn't a fix for this yet--has anyone had luck tweaking settings to get it better?

**Comment by julien-f** [comment]
Just to add to this, I'm having the same issue on the Steam Deck and both the CPU and GPU stay low (~30%) during the issue.
